mkt/extensions/views.py

```diff
@@ -70,8 +70,22 @@ def create(self, request, *args, **kwargs):
         return Response(serializer.data, status=status.HTTP_202_ACCEPTED)
 
     def validate_upload(self, file_obj):
```

Comment:
That would also allow you to remove the weird instantiation of the view class in the tests